src/schemas/json/buf.json

```diff
@@ -0,0 +1,454 @@
+{
+  "$comment": "https://buf.build/docs/configuration/v1/buf-yaml",
+  "$id": "https://json.schemastore.org/buf.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "buf.yaml",
+  "description": "buf.yaml is used to define a module. It's the primary configuration file, and is responsible for the module's name, dependencies, and lint and breaking configuration.",
+  "type": "object",
+  "$defs": {
+    "lint-rule": {
+      "type": "string",
+      "description": "https://buf.build/docs/lint/rules",
+      "enum": [
+        "DEFAULT",
+        "BASIC",
+        "MINIMAL",
+        "COMMENTS",
+        "UNARY_RPC",
+        "DIRECTORY_SAME_PACKAGE",
+        "PACKAGE_DEFINED",
+        "PACKAGE_DIRECTORY_MATCH",
+        "PACKAGE_SAME_DIRECTORY",
+        "ENUM_PASCAL_CASE",
+        "ENUM_VALUE_UPPER_SNAKE_CASE",
+        "FIELD_LOWER_SNAKE_CASE",
+        "MESSAGE_PASCAL_CASE",
+        "ONEOF_LOWER_SNAKE_CASE",
+        "PACKAGE_LOWER_SNAKE_CASE",
+        "RPC_PASCAL_CASE",
+        "SERVICE_PASCAL_CASE",
+        "PACKAGE_SAME_CSHARP_NAMESPACE",
+        "PACKAGE_SAME_GO_PACKAGE",
+        "PACKAGE_SAME_JAVA_MULTIPLE_FILES",
+        "PACKAGE_SAME_JAVA_PACKAGE",
+        "PACKAGE_SAME_PHP_NAMESPACE",
+        "PACKAGE_SAME_RUBY_PACKAGE",
+        "PACKAGE_SAME_SWIFT_PREFIX",
+        "ENUM_FIRST_VALUE_ZERO",
+        "ENUM_NO_ALLOW_ALIAS",
+        "IMPORT_NO_WEAK",
+        "IMPORT_NO_PUBLIC",
+        "IMPORT_USED",
+        "ENUM_VALUE_PREFIX",
+        "ENUM_ZERO_VALUE_SUFFIX",
+        "FILE_LOWER_SNAKE_CASE",
+        "RPC_REQUEST_RESPONSE_UNIQUE",
+        "RPC_REQUEST_STANDARD_NAME",
+        "RPC_RESPONSE_STANDARD_NAME",
+        "PACKAGE_VERSION_SUFFIX",
+        "SERVICE_SUFFIX"
+      ]
+    },
+    "breaking-rule": {
+      "type": "string",
+      "description": "https://buf.build/docs/breaking/rules",
+      "enum": ["FILE", "PACKAGE", "WIRE_JSON", "WIRE"]
+    }
+  },
+  "required": ["version"],
+  "properties": {
+    "version": {
+      "description": "The version key is required, and defines the current configuration version. The only accepted values are v1beta1 and v1.",
+      "type": "string",
+      "enum": ["v1", "v1beta1"]
```

Comment:
@Alfus good point - I didn't do an extensive dig on "what were the valid schema values of `v1beta1`", as it would be somewhat tricky to splice together with our existing docs. I assumed that most of them were, but definitely not all.

I mostly wanted to not error on a `v1beta1` in a config as it's technically a valid value here. I'm not sure if there's a way to mark the just the value as deprecated. It's probably more correct to split this out to a `buf-v1beta1.json`.